### PR TITLE
Add a new VSIX project for Microsoft.CodeAnalysis.FxCopAnalyzers

### PR DESCRIPTION
**Pull request by mavasani**
This is essentially a packaging VSIX which contains the underlying VSIXes for different analyzer packages.

### CODE REVIEW

**Comment by mavasani** [comment]
Mostly copied over from https://github.com/dotnet/roslyn/blob/master/src/Deployment/Roslyn.csproj

**Comment by mavasani** [comment]
Tagging @jinujoseph @333fred @heejaechang for review

**Comment by mavasani** [comment]
It shows up as a single VSIX "Microsoft.CodeAnalysis.FxCop" in `Extensions and Updates` dialog.

File: RoslynAnalyzers.sln [hunk 1]
```diff
@@ -94,6 +94,8 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 		README.md = README.md
 	EndProjectSection
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "FxCopAnalyzersSetup", "src\FxCopAnalyzersSetup\FxCopAnalyzersSetup\FxCopAnalyzersSetup.csproj", "{2638D846-F5D3-4188-AF01-E94331552D2D}"
```
Comment (333fred) [inline RoslynAnalyzers.sln]: All of our other csprojs use the other GUID, can we make them consistent?

Comment (mavasani) [inline RoslynAnalyzers.sln]: Yes, sure.

**Comment by jinujoseph** [comment]
so for the uninstall scenario then need to just uninstall one vsix ?

**Comment by mavasani** [comment]
@jinujoseph - Yes.

**Comment by mavasani** [comment]
@333fred - Made the change you recommended.

**Pull request merged**